README.md

```diff
@@ -77,7 +79,13 @@ Then you can run the tests:
 ```bash
 npm run test-examples
 ```
+warning:
+	WebGl EXT_frag_depth extension not supported in headless mode
 
+option in headless mode:
+	* launch test with remote debbuging on port 9222:
+		- `REMOTE_DEBUGGING=1 npm test`
```

Comment:
You don't need remote debugging. The `DEBUG=1` env var starts puppeteer in a Chrome (with a window).

Reply:
As I said above, there are differents behaviors between headless and headfull (as extention EXT_frag_depth) 
how to debug a test that passes in headfull and not in headless?

Reply:
Ah my bad, I misread the description.